main.rb

```diff
@@ -0,0 +1,41 @@
+require_relative 'app'
+
+APP = App.new
+
+def main
+  puts "\nPlease choose an option by entering a number:
+1 - List all books
+2 - List all people
+3 - Create a person
+4 - Create a book
+5 - Create a rental
+6 - List all rentals for a given person id
+7 - Exit"
+  user_choice = gets.chomp.to_i
+  case user_choice
+  when 1
+    APP.list_of_books
+  when 2
+    APP.list_of_people
+  when 3
+    APP.person_info
+  when 4
+    APP.create_book
+  when 5
+    APP.create_rental
+  when 6
+    APP.list_rentals_by_person_id
```

Comment:
- [ ] **Good job so far 👏 👏** Everything works perfectly, but I didn't get the expected result while listing the `rentals`, Kindly fix this 👇 and try to replicate the result from the [video ](https://www.youtube.com/watch?v=vkkgrhD6aXQ) 😎😎
   ![image](https://user-images.githubusercontent.com/102757126/232187443-565b1a75-cebc-43ae-b4e7-6d4df0547bfa.png)